src/tiledb/cloud/client.py

```diff
@@ -63,6 +63,36 @@ def Ctx(config=None):
     return tiledb.Ctx(Config(config))
 
 
+class AWS(enum.Enum):
+    """
+    The AWS cloud regions that are currently supported.
+
+    US_EAST_1 = North America, United States (Virginia)
+    US_WEST_1 = North America, United States (California)
+    EU_WEST_1 = Europe, Ireland
+    EU_WEST_2 = Europe, London
+    AP_SOUTHEAST_1 = Asia, Singapore
+    """
+
+    US_EAST_1 = "https://us-east-1.aws.api.tiledb.com"
```

Comment:
Maybe we can expose a method to the clients to just print all the available regions?

Reply:
My expectation was that users turn to the docs to see what regions are supported. That said, if you see value in such a function then I'm happy to add it.